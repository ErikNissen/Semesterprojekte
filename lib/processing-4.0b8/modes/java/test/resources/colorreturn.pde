version https://git-lfs.github.com/spec/v1
oid sha256:5926b1b351e3be09237db83f8e2740ad329ecabb87aee4929e85d9e0a6f086fc
size 190
