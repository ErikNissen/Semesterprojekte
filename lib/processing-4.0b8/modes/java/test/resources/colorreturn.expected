version https://git-lfs.github.com/spec/v1
oid sha256:fffb768be5df51f90653d1a69db74d689359c39eaab02cd16eacd5440103722f
size 923
